planet-clicker/Assets/_Script/Action/AddCount.cs

```diff
@@ -12,46 +12,39 @@ namespace _Script.Action
     public class AddCount : ActionBase
     {
         // 내부적으로 count 값을 관리하기 위한 변수
-        private CountData _count;
-
-        public AddCount()
-        {
-        }
+        private CountData _data;
 
         public AddCount(long count)
         {
-            _count = new CountData(count);
+            _data.Sum(count);
```

Comment:
I think this should be `_data = new CountData(count)`?

Reply:
perfect point, and modified